### ISSUE TITLE
ffmpeg build fails

### ISSUE THREAD

**Issue by Selur** [jb-alvarado/media-autobuild_suite#154]
with:
```
libavdevice/decklink_common.cpp:24:27: fatal error: DeckLinkAPI_i.c: No such file or directory
compilation terminated.
common.mak:60: recipe for target 'libavdevice/decklink_common.o' failed
make: *** [libavdevice/decklink_common.o] Error 1
```
see: https://pastee.org/9v4x7

**Comment by wiiaboo** [comment]
What does `find /local* -name DeckLinkAPI*` in mintty say?

**Comment by Selur** [comment]
```
$ find /local* -name DeckLinkAPI*
/local32/include/DeckLinkAPI.h
/local32/include/DeckLinkAPIVersion.h
/local64/include/DeckLinkAPI.h
/local64/include/DeckLinkAPIVersion.h
/local64/include/DeckLinkAPI_i.c
```

**Comment by wiiaboo** [comment]
So for some reason it didn't download for /local32. Maybe github timed out for that. Just copy /local64 files to /local32.

**Comment by Selur** [comment]
wouldn't that also overwrite the binaries?!?

**Comment by wiiaboo** [comment]
The `DeckLinkAPI*` files from `/local64/include/` to `/local32/include/`.
What do you mean, binaries?

**Comment by wiiaboo** [comment]
Probably fixed with 6346ef6f58603aeb1f97ee368c4aabce9aea21a6

**Comment by Selur** [comment]
ah, okay sounded like you wanted me to copy the whole local64 to the local32 folder. :)
+ you are right seems like github is once again making problems
```
fatal: unable to access 'https://github.com/rocky/libcdio-paranoia.git/': Connection timed out after 300031 milliseconds
error: Could not fetch origin
```
-> I'll probably best wait untill github is more responsive
